pytest_splunk_addon/standard_lib/cim_tests/test_templates.py

```diff
@@ -18,4 +22,31 @@ class CIMTestTemplates(object):
     """
     logger = logging.getLogger("pytest-splunk-addon-cim-tests")
 
-    # TODO: Add Test scenarios 
+    @pytest.mark.parametrize(
+        "app_name", [pytest.param("Splunk_SA_CIM", marks=[pytest.mark.searchtime_cim])]
```

Comment:
Update the marker to "splunk_app_cim".

Reply:
As discussed, we need to update the marker to "splunk_searchtime_cim"

Reply:
Updated marker name